src/lib.c

```diff
@@ -16,9 +16,9 @@
 
 #include "efivar.h"
 
-#define EFI_VARTOFILE	 "b2ac5fc9-92b7-4acd-aeac-11e818c3130c-VarToFile"
+#define VAR2FILE	"/sys/firmware/efi/efivars/VarToFile-b2ac5fc9-92b7-4acd-aeac-11e818c3130c"
```

Comment:
This needs to be squashed with the the patch that introduced the change

Reply:
Done